src/App.js

```diff
@@ -48,28 +65,40 @@ function App() {
   );
 }
 
+/*  function decodeString(str) {
+      const textArea = document.createElement('textarea')
+      textArea.innerHTML= str
+      return textArea.value
+  }
+
+  function handleSubmit(e) {
+      e.preventDefault()
+      axios
+          .get(FlashcardData, {
+              params: {
+                  amount: amountEl.current.value,
+                  language: languageEl.current.value
+              }
+          })
+          .then(res => {
+              setFlashcards(res.data.results.map((questionItem, index) => {
+                  const answer = decodeString(questionItem)
+                  return {
+                      id: `${index}-${Date.now()}`,
+                      question: decodeString(questionItem.question),
+                      answer: answer
+                  }
+              }))
+          })
```

Comment:
You are using version control (git). Remove the unused comments. You have them in the previous revision if you need to copy them or revert them. 

The comments are for messages that can not be read in the code (ex: why something is done)